<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration Editing Tool </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="jobView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mainMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuJobs">
    <property name="title">
     <string>Add new Job</string>
    </property>
    <addaction name="actionWake"/>
    <addaction name="actionShutdown"/>
    <addaction name="actionDisk_space_check"/>
    <addaction name="actionBackup"/>
    <addaction name="actionConsole"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuJobs"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionWake">
   <property name="text">
    <string>Wake</string>
   </property>
  </action>
  <action name="actionShutdown">
   <property name="text">
    <string>Shutdown</string>
   </property>
  </action>
  <action name="actionDisk_space_check">
   <property name="text">
    <string>Disk space check</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="text">
    <string>Console</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
